lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -442,6 +451,7 @@ def main():
             module.fail_json(msg="Name of ignored table cannot be empty")
     single_transaction = module.params["single_transaction"]
     quick = module.params["quick"]
+    no_create_db = module.params["no_create_db"]
```

Comment:
you also need to pass the flag to the mysql_dump function where it's invoked in the code

Reply:
you also need to pass the flag to the mysql_dump function where it's invoked in the code